Documentation/articles/content/why_content_pipeline.md

```diff
@@ -0,0 +1,69 @@
+# Why use the Content Pipeline
+
+The MonoGame team has been putting a lot of effort into a cross-platform content pipeline, but would you use the **Content Pipeline** when MonoGame also supports loading assets natively, like .png, .mp3, .wav? Well, it all boils down to a couple of words, performance and efficiency.
+
+## Textures
+
+Most texture formats (like PNGs) are generally not optimized for games.
+
+For example, most GPUs have specific hardware supported compressed formats for textures. PNGs or JPEGs just get uncompressed when passed to the GPU. By using the hardware compression you often get 4x to 8x more textures space for your game as well as faster load times and smaller packages.
+
+If we have a 256×256 32 bit .png texture that we are using in our game, why should a developer bother with compiling to .xnb's, they just want to use the texture as a raw .png file. On disk .png is very impressive in its size, that image probably only takes up 2-5 kb on disk, keeping your application package size down. Great!
+
+However, what happens when we load this .png from storage onto a device at runtime (like an iPhone).  The texture is loaded from storage into memory and decompressed/unpacked from its compressed png format into raw bytes. This is done because the GPU on your device does not know how to use a png image directly, it can only use certain types of compression. With the image loaded into memory, this takes up 262,144 bytes (256x256x4, the x4 is because we have 1 byte per channel Red, Green, Blue, and Alpha).
```

Comment:
> (like an iPhone)

PNGs are decompressed on all platforms... no need to give an example of a platform.

Reply:
>  This is done because the GPU on your device does not know how to use a png image directly,

Didn't we already say this in the first paragraph of this section?

Reply:
resolved points above